src/zenml/zen_stores/migrations/versions/6f707b385dc1_fix_model_artifacts.py

```diff
@@ -10,7 +10,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "6f707b385dc1"
-down_revision = "0.45.0"
+down_revision = "7f603e583dd7"
 branch_labels = None
```

Comment:
@bcdurak changing this in the upgrade tree seems very wrong to me, I think this is rewriting history in a way that might break things for people who are running on 0.45.0 now. Is this file somethign that was merged into develop after the 0.45.0 release?

Reply:
ah went to investigate, all good with this.